en/simulation/README.md

```diff
@@ -225,14 +225,7 @@ For more information about using the MAVLink module see [Modules Reference: Comm
 
 With Hardware-in-the-Loop (HITL) simulation the normal PX firmware is run on real hardware. *QGroundControl* is connected to the physical hardware over USB and acts as a gateway to forward data between the simulator, PX4 and any offboard API.
 
-The diagram below shows a typical HITL simulation environment:
```

Comment:
FYI makes sense to move this content to HITL doc and just link from here because it is relevant to understanding how HITL is set up.